source/picotron.lua

```diff
@@ -0,0 +1,51 @@
+--[[pod_format="raw",created="2024-10-14 18:50:00",modified="2024-10-14 19:29:59",revision=136]]
+if not import then 
+  import = function(path) 
+    new_path, _ = string.gsub(path, "%.%./", "")
+    return load(fetch("tinta/source/" .. new_path .. ".lua"))()
+  end 
+end
```

Comment:
had to add a small polyfill for import that removes relative paths since picotron likes absolute paths from the cart directory

this means:

1. removing `../` prefixes
2. adding `tinta/source/` as a prefix
3. adding `.lua` as a suffix

this is to make sure that all lookups in `/ram/cart` happen correctly, all while not having to edit any of the existing code✨